test/libsolidity/smtCheckerTests/operators/bitwise_operators_do_not_throw_exceptions.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.0;
```

Comment:
We don't include these in tests explicitly. Our test framework includes them automatically.

Reply:
@pgebal these lines above should be removed